packages/lesmis/src/routes/AdminPanelApp.js

```diff
@@ -181,12 +184,10 @@ const AdminPanelApp = ({ user, isBESAdmin }) => {
         <LogoutPage redirectTo={`${adminUrl}/login`} />
       </Route>
       <LesmisAdminRoute path={`${path}/viz-builder`} isBESAdmin>
-        <VizBuilderProviders>
-          <VizBuilderApp
-            basePath={adminUrl}
-            Navbar={({ user: vizBuilderUser }) => <AdminPanelNavbar user={vizBuilderUser} />}
-          />
-        </VizBuilderProviders>
+        <VizBuilderApp
+          basePath={adminUrl}
+          Navbar={({ user: vizBuilderUser }) => <AdminPanelNavbar user={vizBuilderUser} />}
```

Comment:
I think we'll need to be mindful of this for LESMIS. It's the correct change, but we may need to give the LESMIS Admin users `VIZ_BUILDER_USER` access. Can you add a merge task around that @chris-pollard?

Reply:
Yep! @rohan-bes  Where would I record a merge task? Is this our prod merge (task in release notes) or merging into external LESMIS repo?

Reply:
Yeah just in the release notes should be okay. Just flag that we need to release this change to LESMIS.

@IgorNadj how does adding the steps to do a LESMIS release to the release process slab doc sound to you? It'd be nice if this was a job that the release manager should be responsible for

Reply:
Ah yeah it would be nice, but a LESMIS release would require comms and planning and potentially take weeks, so we can't really put it into a process. 

When we do our next LESMIS release in a couple months we will have to go back through all previous releases and look at the release notes, so if we highlight things that will affect LESMIS in the Release notes that would be helpful. E.g. this one could be "Viz Builder Access Separated From BES Admin (admin-panel and LESMIS)" or something like that.